app/assets/stylesheets/application.scss

```diff
@@ -205,17 +194,33 @@ select{padding: initial;}
   background-image: -webkit-linear-gradient(top, whiteSmoke 0%, #EEE 100%);
   background-image: -ms-linear-gradient(top, whiteSmoke 0%, #EEE 100%);
   background-image: -o-linear-gradient(top, whiteSmoke 0%, #EEE 100%);
-  filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#f5f5f5', endColorstr='#eeeeee',GradientType=0 );
+  filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#f5f5f5', endColorstr='#eeeeee', GradientType=0);
```

Comment:
I would drop all vendor prefixes here except for webkit, since [both opera and FF support the non-prefixed version for at least the last 15 versions](http://caniuse.com/#feat=css-gradients). I agree that we don't need to provide the filter either for old IE versions.  I also agree we shoul provide a solid background fallback for anyone using an antique browser.